File: nomorescript-plugin/src/test/resources/sandbox/method.scala
class MethodTest {
  def method1(param1: String) = {
    println(param1)
    10
  }

  def method2(param2: String) = {
    method1(param2)
  }

  def method3(param3: String = "hello") = {
    method2(param3)
  }

  method3()
}

File: nomorescript-plugin/src/test/resources/sandbox/method.js
MethodTest = function () {
    this.method3("hello");
};

MethodTest.prototype.method1 = function (param1) {
    println(param1)
    10
}

MethodTest.prototype.method2 = function (param2) {
    method1(param2)
}

MethodTest.prototype.method3 = function (param3) {
    method2(param3)
}
